docs/src/components/HeadSEO.astro

```diff
@@ -3,10 +3,15 @@ import {SITE, OPEN_GRAPH} from '../config.ts';
 export interface Props {
   content: any,
   site: any,
+  formatTitle(content: any, SITE: any): string,
```

Comment:
I'd like to avoid a custom function if we can do this via a normal string prop instead. It seems like both `HeadSEO` and the `<title>` component need the same title value, so this feels like a value that should be created outside of this component and passed in as a prop instead. Especially since this formatTitle function seems to get defined in a few places. 

What about this API instead?

1. HeadSEO takes required `title` Prop. In the future, maybe site title should be some global Astro config value, in which case `title` could be an optional prop.
2. `MainLayout` creates our custom title as a variable: 
```
const pageTitle = content.title ? `${content.title} 🚀 ${SITE.title}` : SITE.title;
```
3. `MainLayout` passes the variable to `<HeadSEO>` & `<title>`:

```
<HeadSEO {content} canonicalURL={Astro.request.canonicalURL} title={pageTitle} />
<title>{pageTitle}</title>
```